doc/changes/changes_2.0.8.md

```diff
@@ -0,0 +1,16 @@
+# Spark Connector Common Java 2.0.8, released 2024-09-24
```

Comment:
Did you intentionally set tomorrow as release date to have some buffer? :-)

Reply:
Yes, sometimes PK goes wild and require lots of fixing/dancing around. So, having extra day for the release normally a good idea